sonic-net/sonic-buildimage
# [armhf][Nokia-7215]Add SFP refactor support for Nokia-7215 platform

#### Why I did it

Add support for SFP refactor on Nokia-7215 Marvell armhf platform.
    
    Platform: armhf-nokia_ixs7215_52x-r0
    HwSKU: Nokia-7215
    ASIC: marvell
    Port Config: 48x1G + 4x10G (SFP+)

#### How I did it

    Modify sfp.py to support SFP refactor optoe driver and platform.json to facilitate proper OC test completion.

#### How to verify it

    Build armhf target for Nokia-7215 and verify proper Xcvrd and SFP refactor operation.

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
     
     [armhf][Nokia-7215]Add SFP refactor support for Nokia-7215 platform
